src/views/AllContacts.vue

```diff
@@ -1,5 +1,121 @@
 <template>
-  <div class="about">
-    <h1>This is a list page</h1>
-  </div>
+  <v-container fluid id="containerAll">
+    <v-text-field
+      v-model="search"
+      append-icon="mdi-magnify"
+      label="Search for a contact"
+      hide-details
+    ></v-text-field>
+    
+    <Pagination
+      v-if="tableData"
+      v-model="pagination"
+      :totalRecords="tableData.length"
+      :perPageOptions="perPageOptions"
+    />
+
+    //This table it's working only with search bar
+    <v-data-table
+      :headers="config"
+      :items="tableData"
+      :search="search"
+      :items-per-page = "perPageOptions"
+      :pagination="pagination"
+    ></v-data-table>
+
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      <v-divider></v-divider>
+      
+    //This table it's working only with pagination
```

Comment:
Make only one table to work with the search field and with the pagination component too.
Else, delete the pagination component and this table that work with this pagination component, and leave only the first table, which has default pagination, that works very well.